client/components/About.js

```diff
@@ -10,14 +10,19 @@ class About extends React.Component {
       <div className='about'>
         <Header />
         <div className='container-fluid animated fadeIn'>
+          <div className='row text-center'>
+            <h3>About the Project</h3>
+          </div>
           <img className='group-photo' src={groupPhoto} />

```

Comment:
Please move the image as suggested by @goldbergtatyana here https://github.com/MusicConnectionMachine/VisualizationG6/issues/88#issuecomment-296456810